client/deis.py

```diff
@@ -378,6 +378,27 @@ def __init__(self):
         self._settings = Settings()
         self._logger = logging.getLogger(__name__)
 
+    def check_connection(self, controller, ssl_verify):
+        url = urlparse.urljoin(controller, '/v1/')
+
+        response = self._session.get(url, allow_redirects=False,
+                                     verify=ssl_verify)
+
+        if response.status_code != 401:
+            raise EnvironmentError("{} is not a valid deis controller url.".format(controller))
+        self.check_api_version(response.headers.get('X_DEIS_API_VERSION'))
+
+    def check_api_version(self, server_api_version):
+        """
+        Check if the client is compatable with the api
+        """
+        if server_api_version is not None and server_api_version != __api_version__:
+            self._logger.warning("""
+         !    WARNING: Client and server API versions do not match. Please consider upgrading.
```

Comment:
since this is in a quoted string, the whitespace is important. Could you please shift this back to the same format as it used to be?